Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -910,6 +909,12 @@ namespace xr
             depthInfoView.subImage.imageRect = imageRect;
             depthInfoView.subImage.imageArrayIndex = 0;
         }
+
+        bool IsFrameSubmissionSupported()
+        {
+            const auto& sessionState = sessionImpl.HmdImpl.Context.State();
+            return sessionState != XrSessionState::XR_SESSION_STATE_IDLE;
```

Comment:
Should this be `return sessionState > XrSessionState::XR_SESSION_STATE_IDLE && sessionState < XrSessionState::XR_SESSION_STATE_STOPPING;`?

Reply:
Chatted with OpenXR owners, sounds like they'd actually suggest creating a flag based on whether we have called xrBeginSession/xrEndSession. I'll update this now